lib/elixir/unicode/unicode.ex

```diff
@@ -1,15 +1,18 @@
 # How to update the Unicode files
 #
-# Unicode files can be found in https://www.unicode.org/Public/
+# Unicode files can be found in https://www.unicode.org/Public/VERSION_NUMBER/ where
+# VERSION_NUMBER is the current Unicode version.
 #
 # 1. Replace UnicodeData.txt by copying original
-# 2. Replace PropList.txt by copying original
-# 3. Replace SpecialCasing.txt by copying original and removing conditional mappings
-# 4. Replace IdentifierType.txt by copying original (from /Public/security)
-# 5. Replace confusables.txt by copying original (from /Public/security)
-# 6. Replace PropertyValueAliases.txt by copying original
-# 7. Replace Scripts.txt by copying original
-# 8. Replace ScriptExtensions.txt by copying original
+# 2. Replace PropertyValueAliases.txt by copying original
+# 3. Replace PropList.txt by copying original
+# 4. Replace ScriptExtensions.txt by copying original
+# 5. Replace Scripts.txt by copying original
+# 6. Replace SpecialCasing.txt by copying original and removing conditional mappings
```

Comment:
Hi @eksperimental! I assume that you automated the "removing conditional mappings" part, correct? Perhaps we should add the part that ignores the conditional mappings to Elixir instead?

Reply:
Yes, I can do it from Elixir, I pushed a commit here: https://github.com/elixir-lang/elixir/commit/18c44e2e4fb6cfe00f04eb0117f648bb57227c8c

So it should be simpler for you to copy this file now!

Btw, Unicode 15.0 is out, so if you want to trigger your automation soon, that would be nice!

Reply:
> Hi @eksperimental! I assume that you automated the "removing conditional mappings" part, correct? Perhaps we should add the part that ignores the conditional mappings to Elixir instead?

Hi,
Yes I already taking care of it.
https://github.com/eksperimental/maintenance/blob/0a9acb151f95c98ea5fd27037c12ff9056324eca/apps/maintenance/lib/maintenance_job/unicode.ex#L131-L149

Reply:
> Btw, Unicode 15.0 is out, so if you want to trigger your automation soon, that would be nice!

New MAJOR Unicode versions of  come out in September.
The app is already running (well, with the outdate version before the latest changes). The app checks every 6 hours for a new Unicode release (any new version), so you will see a PR the same day it comes out.
If there is no new PR let me know because something went wrong.

Reply:
> Btw, Unicode 15.0 is out, so if you want to trigger your automation soon, that would be nice!

Forgot to say. What you see in the Public/ dir is an alpha-review of 15.0.0

Reply:
> Forgot to say. What you see in the Public/ dir is an alpha-review of 15.0.0

Ah, this makes perfect sense, thanks!